<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartProfileDialog</class>
 <widget class="QDialog" name="StartProfileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>186</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Start profiling</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="TextLabel1">
       <property name="text">
        <string>Command&amp;line:</string>
       </property>
       <property name="buddy">
        <cstring>cmdlineCombo</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="cmdlineCombo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Enter the commandline parameters</string>
       </property>
       <property name="whatsThis">
        <string>&lt;b&gt;Commandline&lt;/b&gt;
&lt;p&gt;Enter the commandline parameters in this field.&lt;/p&gt;</string>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="insertPolicy">
        <enum>QComboBox::InsertAtTop</enum>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToMinimumContentsLengthWithIcon</enum>
       </property>
       <property name="duplicatesEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="TextLabel2">
       <property name="text">
        <string>&amp;Working directory:</string>
       </property>
       <property name="buddy">
        <cstring>workdirPicker</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="E5ComboPathPicker" name="workdirPicker" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="focusPolicy">
        <enum>Qt::WheelFocus</enum>
       </property>
       <property name="toolTip">
        <string>Enter the working directory</string>
       </property>
       <property name="whatsThis">
        <string>&lt;b&gt;Working directory&lt;/b&gt;
&lt;p&gt;Enter the working directory of the application to be debugged. Leave it empty to set the working directory to the executable directory.&lt;/p&gt;</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="textLabel1">
       <property name="text">
        <string>&amp;Environment:</string>
       </property>
       <property name="buddy">
        <cstring>environmentCombo</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="environmentCombo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Enter the environment variables to be set.</string>
       </property>
       <property name="whatsThis">
        <string>&lt;b&gt;Environment&lt;/b&gt;
&lt;p&gt;Enter the environment variables to be set for the program. The individual settings must be separated by whitespace and be given in the form 'var=value'. In order to add to an environment variable, enter it in the form 'var+=value'.&lt;/p&gt;
&lt;p&gt;Example: var1=1 var2=&quot;hello world&quot; var3+=&quot;:/tmp&quot;&lt;/p&gt;</string>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="insertPolicy">
        <enum>QComboBox::InsertAtTop</enum>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToMinimumContentsLengthWithIcon</enum>
       </property>
       <property name="duplicatesEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QCheckBox" name="exceptionCheckBox">
       <property name="toolTip">
        <string>Uncheck to disable exception reporting</string>
       </property>
       <property name="whatsThis">
        <string>&lt;b&gt;Report exceptions&lt;/b&gt;
&lt;p&gt;Uncheck this in order to disable exception reporting.&lt;/p&gt;</string>
       </property>
       <property name="text">
        <string>Report &amp;exceptions</string>
       </property>
       <property name="shortcut">
        <string>Alt+E</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="clearShellCheckBox">
       <property name="toolTip">
        <string>Select to clear the display of the interpreter window</string>
       </property>
       <property name="whatsThis">
        <string>&lt;b&gt;Clear interpreter window&lt;/b&gt;&lt;p&gt;This clears the display of the interpreter window before starting the debug client.&lt;/p&gt;</string>
       </property>
       <property name="text">
        <string>Clear &amp;interpreter window</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="consoleCheckBox">
       <property name="toolTip">
        <string>Select to start the debugger in a console window</string>
       </property>
       <property name="whatsThis">
        <string>&lt;b&gt;Start in console&lt;/b&gt;
&lt;p&gt;Select to start the debugger in a console window. The console command has to be configured on the Debugger-&amp;gt;General page&lt;/p&gt;</string>
       </property>
       <property name="text">
        <string>Start in console</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="eraseCheckBox">
       <property name="toolTip">
        <string>Select this to erase the collected timing data</string>
       </property>
       <property name="whatsThis">
        <string>&lt;b&gt;Erase timing data&lt;/b&gt;
&lt;p&gt;Select this to erase the collected timing data before the next profiling run.&lt;/p&gt;</string>
       </property>
       <property name="text">
        <string>Erase &amp;timing data</string>
       </property>
       <property name="shortcut">
        <string>Alt+C</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>E5ComboPathPicker</class>
   <extends>QWidget</extends>
   <header>E5Gui/E5PathPicker.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cmdlineCombo</tabstop>
  <tabstop>workdirPicker</tabstop>
  <tabstop>environmentCombo</tabstop>
  <tabstop>exceptionCheckBox</tabstop>
  <tabstop>clearShellCheckBox</tabstop>
  <tabstop>consoleCheckBox</tabstop>
  <tabstop>eraseCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>StartProfileDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>23</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>StartProfileDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>116</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>116</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
